### PR TITLE
fix linking errors

### PR DESCRIPTION
**Pull request by norihiro-w**
I encountered some linking errors with the latest maser and fixed them with this PR. However, travis and Jenkins are actually working fine. so it could be specific to my environment.

### CODE REVIEW

File: MeshGeoToolsLib/CMakeLists.txt [hunk 1]
```diff
@@ -17,6 +17,7 @@ target_link_libraries(MeshGeoToolsLib INTERFACE
 	MathLib
 	MeshLib
 	GeoLib
+	FileIO
```
Comment (norihiro-w) [inline MeshGeoToolsLib/CMakeLists.txt]: GeoMapper class uses FileIO

File: MeshLib/CMakeLists.txt [hunk 2]
```diff
@@ -21,6 +21,7 @@ target_link_libraries(MeshLib INTERFACE
 	BaseLib
 	GeoLib
 	MathLib
+	${CATALYST_LIBRARIES}
```
Comment (norihiro-w) [inline MeshLib/CMakeLists.txt]: vtkMeshConverter needs this

File: AssemblerLib/CMakeLists.txt [hunk 3]
```diff
@@ -13,6 +13,10 @@ include_directories(
 	../MeshLib
 )
 
+target_link_libraries(AssemblerLib INTERFACE
```
Comment (norihiro-w) [inline AssemblerLib/CMakeLists.txt]: Jenkins doesn't but travis requires this

**Comment by endJunction** [comment]
Do not mind the ogsbot. I have broken the Linux-PRs jenkins config before, and now it tries to recompile all pending PRs. Sorry.

**Pull request merged**